api/src/main/java/com/cloud/user/UserAccount.java

```diff
@@ -67,4 +67,18 @@ public interface UserAccount extends InternalIdentity {
     public String getExternalEntity();
 
     public void setExternalEntity(String entity);
+
+    public boolean isTwoFactorAuthenticationEnabled();
+
+    public void setTwoFactorAuthenticationEnabled(boolean twoFactorAuthenticationEnabled);
+
+    public String getKeyFor2fa();
+
+    public void setKeyFor2fa(String keyFor2fa);
+
+    public String getUser2faProvider();
+
+    public void setUser2faProvider(String user2faProvider);
```

Comment:
There are instances of `TwoFactorAuthentication` and `2fa` in this and others files as well. Maybe standardize the name to `2fa` as it is more concise.

Reply:
Expect for the provider names, APIs and responses, I've now changed the usage to 2fa.